[skip ci] Document '[skip ci]'

driazati opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#55418 [skip ci] Document '[skip ci]'**

This documents the functionality to skip CI as discussed in #55174

Differential Revision: [D27609269](https://our.internmc.facebook.com/intern/diff/D27609269)

Commit:
[Skip CI] Document '[Skip CI]'

[ghstack-poisoned]

Commit:
Update on "[Skip CI] Document '[Skip CI]'"




[ghstack-poisoned]

janeyx99 commented on CONTRIBUTING.md:
```diff
@@ -138,6 +138,8 @@ and `python setup.py clean`. Then you can install in `develop` mode again.
 with `brew install cmake` if you are developing on MacOS or Linux system.
 * Our `setup.py` requires Python >= 3.6
+* If a commit is simple and doesn't affect any code (keep in mind that some docstrings contain code
+  that is used in tests), you can add `[skip ci]` to your commit message to skip all build / test steps.
```
Is it only in the commit message? Could one have the [skip ci] in the PR title or body as well?

janeyx99 replied:
I also see you experimenting with the capitalization. Maybe include a note about the casing if it's restricting?

janeyx99 replied:
Ah I see it has to be the commit message: do you think it may be helpful to link https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/?

driazati replied:
Based on #55425 looks like anywhere in the commit message (subject or body) is fine, but updating the PR's title/body on GitHub has no effect: #55426

driazati committed:
Update on "[skip ci] Document '[skip ci]'"




[ghstack-poisoned]

driazati committed:
Update on "[skip ci] Document '[skip ci]'"




[ghstack-poisoned]

driazati committed:
Update on "[skip ci] Document '[skip ci]'"


This documents the functionality to skip CI as discussed in #55174

Differential Revision: [D27609269](https://our.internmc.facebook.com/intern/diff/D27609269)

[ghstack-poisoned]
